core-ui/src/main/java/com/maxshop/fragment/BaseFragment.kt

```diff
@@ -1,16 +1,16 @@
 package com.maxshop.fragment
 
 import android.widget.Toast
-import com.maxshop.viewModel.MultiViewModuleFactory
+import com.maxshop.viewModel.ViewModuleFactory
 import dagger.android.support.DaggerFragment
 import javax.inject.Inject
 
 abstract class BaseFragment : DaggerFragment() {
 
     @Inject
-    protected lateinit var factory: MultiViewModuleFactory
+    protected lateinit var factory: ViewModuleFactory
```

Comment:
There is a typo. `ViewModelFactory` not module factory

Reply:
fixed